src/interpreter/bytecode-generator.cc

```diff
@@ -3010,12 +3013,19 @@ void BytecodeGenerator::VisitConditional(Conditional* expr) {
   } else {
     VisitForTest(expr->condition(), conditional_builder.then_labels(),
                  conditional_builder.else_labels(), TestFallthrough::kThen);
-
     conditional_builder.Then();
+
+    builder()->RecordReplayInstrumentation("breakpoint",
+                                           expr->then_expression()->position());
+
     VisitForAccumulatorValue(expr->then_expression());
     conditional_builder.JumpToEnd();
 
     conditional_builder.Else();
+
+    builder()->RecordReplayInstrumentation("breakpoint",
+                                           expr->else_expression()->position());
```

Comment:
This change doesn't seem right.  We don't want to have breakpoints on every ternary subexpression like "a ? 2 : 3", that seems like overkill.  Can we restrict this to ternaries which have a call subexpression, like in the associated issue?  Ideally we would have breakpoints on every call but doing that without overlapping the existing breakpoints might be tricky to implement.

Reply:
As per our discussion: I added `if`s to prevent too many breakpoints. While it is desirable to have a much more complete picture of breakpoints, that might be too involved for our current needs.